CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+* [\#11034](https://github.com/cosmos/cosmos-sdk/pull/11034) Add group Config including the `MaxMetadataLen` to group keeper.
```

Comment:
I don't think it's needed here since the group module hasn't even been released yet